unified-installer/astra-unified-installer.sh

```diff
@@ -1851,21 +1853,25 @@ step_generate_astra_connector_yaml() {
 
     # Default memory limit
     memory_limit=2
-    snapshot_count=""
-    if prompt_user_yes_no "Do you anticipate having more than 10,000 snapshots and backups existing at the same time at any point? "; then
-        prompt_user_number_greater_than_zero snapshot_count "Please estimate the maximum number of snapshots and backups you expect to have existing simultaneously within this cluster? (enter number value): "
-        # Calculate estimated_memory and round up to the nearest integer
-        estimated_memory=$(echo "$snapshot_count 5000" | awk '{printf("%d\n", ($1/$2)+0.6)}')
-
-        # If estimated_memory is greater than memory_limit, set memory_limit to estimated_memory
-        if (( estimated_memory > memory_limit )); then
-            memory_limit=$estimated_memory
-        fi
+    if [  "$DISABLE_PROMPTS" != "true" ]; then
```

Comment:
This fixes the div by 0. 
'prompt_user_yes_no' below assumes the answer is always 'yes' when prompts are disabled.